### ISSUE TITLE
Docs: show how to include dividers in the siteNav

### ISSUE THREAD

**Issue by damithc** [MarkBind/markbind#543]
v1.15.2

Suggestion: provide a way to have dividers in the siteNav. An example (from Hexo):

![image](https://user-images.githubusercontent.com/1673303/50720951-288d7500-10f2-11e9-9b6a-9e326aa0766d.png)

**Comment by Chng-Zhi-Xuan** [comment]
```HTML
<navigation>
  * Getting Started
  * [Home :glyphicon-home:]({{baseUrl}}/index.html)
  * <br></br>
  * <hr></hr>
  * Basic Usage
  * [Home :glyphicon-home:]({{baseUrl}}/index.html)
</navigation>
```

It is already supported, just tested with the above template 😀.

**Comment by yamgent** [comment]
> Key is to use `<br>` and `<hr>` as a list item.

This also works (`<br>` need not be a list item, and we can use Markdown's horizontal bar)

```
<navigation>
  * Getting Started
  * [Home :glyphicon-home:]({{baseUrl}}/index.html)
  <br><br>
  * ------
  * Basic Usage
  * [Home :glyphicon-home:]({{baseUrl}}/index.html)
</navigation>
```

Should we document it as part of tips and tricks?

**Comment by Chng-Zhi-Xuan** [comment]
The since the `<navigation>` area is assumed to only contain list items (by `Page.js#formatSiteNav`), I would prefer not to have HTML tags separating the list items. I think your workaround works because `Markdown-It` treats the `<br>` tag as part of the previous line?

As for documentation, we can seamlessly add it to the `site-nav` example within `Tweaking the page structure` section which I think fits better than `Tips & Tricks` section.

**Comment by damithc** [comment]
Nice find. This worked too:
```
* [Object-Oriented Programming]({{ baseUrl }}/programming/toc/oop.html)
* ------------------------------------------------------------------------
* <br>%%**Appendices**%%<br>
* [Appendix A: Installing 3rd Party Modules]({{ baseUrl }}/programming/toc/thirdparty.html)
```
![image](https://user-images.githubusercontent.com/1673303/53856112-9d563000-400b-11e9-8f4a-af0c98830d2d.png)


> Should we document it as part of tips and tricks?

Can even be a part of the main documentation of site-nav

**Comment by acjh** [comment]
Do we render separate lists around the divider? Can someone help to check the HTML of the Hexo example?

**Comment by acjh** [comment]
Hexo doesn't use a list, but it's one `<div>`:
```html
<div class="inner">
  <strong class="sidebar-title">Getting Started</strong>
  <a href="/docs/" class="sidebar-link current">Overview</a>
  <a href="/docs/setup" class="sidebar-link">Setup</a>
  <a href="/docs/configuration" class="sidebar-link">Configuration</a>
  <a href="/docs/commands" class="sidebar-link">Commands</a>
  <a href="/docs/migration" class="sidebar-link">Migration</a>
  <strong class="sidebar-title">Basic Usage</strong>
  <a href="/docs/writing" class="sidebar-link">Writing</a>
  ...
</div>
```

Let's show an example that keeps it to one list.

**Comment by Tejas2805** [comment]
![Screen Shot 2020-02-27 at 3 48 57 AM](https://user-images.githubusercontent.com/35946746/75381739-46302600-5914-11ea-92ad-57ac6bd431b3.png)

We already have something like this in the `Reader Facing Features` Section ^. Do you all still want to document it separately in siteNav ? Will do that then.

**Comment by damithc** [comment]
> We already have something like this in the `Reader Facing Features` Section. Do you all still want to document it separately in siteNav ? Will do that then.

I think it's better to mention it as an additional tip. In fact, we can also mention how to add a divider to navigation menus too. e.g.,
![image](https://user-images.githubusercontent.com/1673303/75410834-1efb4800-5958-11ea-95c1-81be621305bb.png)

**Comment by Tejas2805** [comment]
Fixed by #1063

**Comment by Tejas2805** [comment]
Can be closed now.
